### ISSUE TITLE
Make blacklist more specific

### ISSUE THREAD

**Issue by g-ortuno** [WebBluetoothCG/registries#19]
Sometimes there are collisions between blacklisted Service UUIDs and non-blacklisted Characteristic UUIDs which results in missing features for Web Bluetooth. For example the FIDO Service (0xfffd) is blacklisted; Playbulb uses that UUID for one of its characteristics which means we can't use the characteristic on Playbulb.

**Comment by jyasskin** [comment]
Playbulb needs to avoid UUIDs in the Bluetooth reserved space or get them officially allocated by the Bluetooth SIG.

**Comment by beaufortfrancois** [comment]
I tend to agree with @g-ortuno there. We should blacklist Service `0xfffd`, not Characteristic `0xfffd` as it is implied in https://github.com/WebBluetoothCG/registries/blob/master/gatt_blocklist.txt
